doc/releases/release-notes-2.1.rst

```diff
@@ -96,6 +96,11 @@ Build and Infrastructure
   Use new functions that utilize eDTS info such as dt_node_reg_addr.
   See :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for details.
 
+* Deprecated direct use of the ``DT_`` Kconfig symbols from the generated
+  ``generated_dts_board.conf``.  We should now utilize either functions from
+  :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for kconfig or functions
+  in zephyr_file:`scripts/sanity_chk/expr_parser.py` for sanitycheck.
```

Comment:
Missing `:`.

Reply:
What if I want to do

if(DT_SOME_DT_OPTION)
endif()

in CMake?

Reply:
Then it should be replaced with a Kconfig option that utilizes a dt function, and than utilize that Kconfig option.

Reply:
This will require a lot of unnecessary boilerplate.

It seems inconsistent to me that C has access to CONFIG_ and DT_ symbols, but CMake has only access to CONFIG_ symbols.

Why should C access the symbols directly and CMake access them indirectly?

I am against this change.

Consistency is good, boilerplate is bad.